src/openslide-vendor-synthetic.c

```diff
@@ -117,15 +117,6 @@ static bool decode_dicom(const void *data, uint32_t len,
     return false;
   }
 
-  // read Data Set metadata and frame
```

Comment:
This line of the comment should be updated, not dropped.

Reply:
The comment could be eg. `// read frame`, but that's the name of the function call, so I wasn't sure it added anything. 

I think the comment was useful when you had to read the metadata before you could read the frame, but that's automatic now with libdicom 1.0, so I thought we could drop it.

Reply:
Yes, except that this function has a comment block at the top of every section, so I'd like to keep that comment to keep the structure skimmable.